pypit/scripts/arcid_plot.py

```diff
@@ -49,10 +49,9 @@ def main(args):
 
     from linetools.utils import loadjson
 
-    from pypit import pyputils
     from pypit import arqa
-
-    msgs = pyputils.get_dummy_logger()
+    from pypit import msgs
+    msgs.reset(verbosity=0)
```

Comment:
Can we set the default to highest verbosity here?

It would address Issue #405